src/main/java/thePackmaster/cards/pinnaclepack/Rea__dyCommonSkill.java

```diff
@@ -0,0 +1,40 @@
+package thePackmaster.cards.pinnaclepack;
+
+import com.evacipated.cardcrawl.mod.stslib.variables.ExhaustiveVariable;
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.actions.common.DrawCardAction;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.powers.DexterityPower;
+import com.megacrit.cardcrawl.powers.LoseDexterityPower;
+import com.megacrit.cardcrawl.powers.watcher.VigorPower;
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class Rea__dyCommonSkill extends AbstractPinnacleCard {
+
+    public final static String ID = makeID("Rea__dyCommonSkill");
+
+    private static final int MAGIC = 3;
+    private static final int MAGIC2 = 2;
+
+    public Rea__dyCommonSkill() {
```

Comment:
*Re__y*
The upgrade on this adds a new effect, which is unusual and makes it harder to players to evaluate the card when they see it. (Again, you can find other examples of this, but the issue with this pack is that there are so many unusual and complex things.) Also, the upgrade both changes Exhaust to Exhaustive and adds this new effect, which is overkill.

The cleanest way to simplify this is to change the upgrade so that it no longer adds Draw 1 card. Instead, it can increase the card's other effects.